src/installer/installer.go

```diff
@@ -429,16 +429,38 @@ func (i *installer) waitForControlPlane(ctx context.Context) error {
 	}
 
 	i.waitForBootkube(ctx)
+	i.waitForETCDBootstrap(ctx)
 
 	// waiting for controller pod to be running
 	if err = i.waitForController(kc); err != nil {
 		i.log.Error(err)
 		return err
 	}
-
 	return nil
 }
 
+func (i *installer) waitForETCDBootstrap(ctx context.Context) {
+	i.UpdateHostInstallProgress(models.HostStageWaitingForBootkube, "waiting for ETCD bootstrap to be complete")
+	i.log.Infof("Started waiting for ETCD bootstrap to complete")
+	for {
```

Comment:
lets use https://github.com/openshift/assisted-installer/blob/master/src/utils/utils.go#L221

Reply:
Thanks Igal! Modified it to use this function, how does it look?